java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -272,6 +272,32 @@ CompletableFuture<Double> zaddIncr(
      */
     CompletableFuture<Map<String, Double>> zpopmax(String key);
 
+    /**
+     * Blocks the connection until removes and returns a member with the highest score from the sorted
+     * sets stored at the specified <code>keys</code>.<br>
+     * <code>BZPOPMAX</code> is the blocking variant of {@link #zpopmax(String)}.<br>
+     * A member with the highest score is popped from first sorted set that is non-empty, with the
```

Comment:
Do you need this sentence after the first one seems like they are explaining the same thing maybe combine the two sentences?